tests/e2e/run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+# This script install dependencies of service side of e2e tests and runs e2e test
+
+# ./run.sh <RTOS connectivity interface>
+# e.g. ./run.sh veth1
+
+if [[ ! -z ${DEBUG_SHELL} ]]
```

Comment:
What is DEBUG_SHELL?  I looked both in our setup readme & general bash instructions and this isn't clear what this is?  

I feel we need a comment here I feel (unless my affinity for PowerShell is just making me totally clueless about this) and if there's special setup needed the readme.md should point user to more.

Reply:
added comment to enable command trace